pyfar/classes/audio.py

```diff
@@ -1501,7 +1503,7 @@ def _matrix_multiplication(a, b, axes, audio_type):
     return np.matmul(a, b, axes=axes)
 
 
-def _match_fft_norm(fft_norm_1, fft_norm_2, division=False):
+def _match_fft_norm(fft_norm_1, fft_norm_2, division=False, n_audio_objects=2):
```

Comment:
```suggestion
def _match_fft_norm(fft_norm_1, fft_norm_2, division=False):
```
then there would be no need to change the behaviour here.